### PR TITLE
Fixed dropdown in nav bar

### PR DESCRIPTION
**Pull request by Wyrd00**
relates to #23 #79

### CODE REVIEW

**Review by tyeung2017 — APPROVED** [review]

**Review by tyeung2017 — APPROVED** [review]

**Review by KatiaAsh — CHANGES_REQUESTED** [review]
drop down should be change

File: landingAr.html [hunk 1]
```diff
@@ -11,17 +11,18 @@
 </head>
 
 <body>
+  <div class="dropdown landing">
+    <button class="dropbtn" id="landingbtn">تغير لغة ↓</button>
+    <div class="landing-content">
+      <a href="landing.html">English</a>
+      <a href="landingAr.html">العربية</a>
```
Comment (KatiaAsh) [inline landingAr.html:18]: in Arabic files
a tag for arabic should be the first

and the same for english file
a tag for english should be the first

Comment (Wyrd00) [reply landingAr.html:18]: Personally, I don't think that really matters

**Review by KatiaAsh — APPROVED** [review]

File: landingAr.html (hunk 1 above)
Comment (KatiaAsh) [reply landingAr.html:18]: It is :v:

**Pull request merged**